### DISCUSSION TITLE
FilamentAsset register an assest that fetch a route

### DISCUSSION THREAD

**Discussion by Jacotheron** [filamentphp/filament#10254] [General]
I have styles that I need to expose both in Filament as well as the front end. These styles are generated by values in a config file, so that we can easily update it. I want to know how I can point the FilamentAsset to this route that will generate the required styles.

I have tried referencing it with the `route` function, but at the time it runs, the route does not exist yet (even moved it to the RouteServiceProvider after the call to read the route files, no luck here). I have added it as an absolute url (without the domain), and now composer fails (due to being unable to copy the file as it don't exist).

A simpler way would have been, being able to directly inject the CSS into a <style> tag for Filament, but I don't see how to easily do this. 

In short, this provides color and font options for the Laraberg editor. Since some of these options are theme based, and the names for these styles are no super intuitive, it don't compile easily with vite.

I have gone with a style tag along with the field content, but it is not ideal (no caching, and duplicated when using more than 1 of these editors on a page).
